documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Word the title such that it explains what is in this doc, e.g. what task is being done (<a href="https://developers.google.com/style/headings#heading-and-title-text" target="_blank">style guide</a>).</p><p> </p><p>The title will be used to list this doc in site navigation, so it needs to be descriptive and something a user wants to click on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>